config/metadata/metadata.yaml

```diff
@@ -30,6 +30,30 @@ rules:
   - apiGroups: ["model.aibrix.ai"]
     resources: ["modeladapters"]
     verbs: ["get", "list"]
+  # For batch job watching
+  - apiGroups: ["batch"] 
+    resources: ["jobs"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  # For batch job ServiceAccount management
+  - apiGroups: [""]
```

Comment:
can we use a fix service account, along with aibrix installation? instead of reply on permission here?

Reply:
em. we can put the file under a separate config/manifest folder without aibrix-system override.  

in helm, I think we can support it as well, something like 

```
  namespace: {{ .Values.job.namespace }}
```